app/controllers/works_controller.rb

```diff
@@ -36,7 +34,7 @@ def wait
   private
 
   def work_params
-    params.expect(work: [:title])
+    params.expect(work: [:title, { related_links_attributes: %i[id text url] }])
```

Comment:
Currently set up to allow only a single related link per work. Done to reduce the surface area of the problem while we vet the overall approach.